internal/bminventory/inventory.go

```diff
@@ -2788,6 +2788,21 @@ func (b *bareMetalInventory) handleReplyError(params installer.V2PostStepReplyPa
 			return err
 		}
 		return b.processDiskSpeedCheckResponse(ctx, h, stepReply, exitCode)
+
+	case models.StepTypeTangConnectivityCheck:
+		// Older agents have no action for tang-connectivity-check.
+		// The missing action result will not fail host validations to
+		// keep backward compatibility with older agents who did not include tang-connectivity-check.
+		missingTangConnectivityActionError := fmt.Sprintf("failed to find action for step type %s", models.StepTypeTangConnectivityCheck)
+
+		if params.Reply.Error == missingTangConnectivityActionError {
```

Comment:
maybe safer to check if the error just contains 'models.StepTypeTangConnectivityCheck' (in case the exact string changes in agent)